docusaurus/docs/getting-started/basic-usage.md

```diff
@@ -7,7 +7,7 @@ sidebar_label: Basic usage
 #### 1. Import `NgxFormObjectModule`
 To start using **ngx-form-object** you have to import `NgxFormObjectModule` into the root module of your project.
```

Comment:
I think this whole step is not needed. You've described in the installation part that this has to be done, it is pretty standard process.

Reply:
It was removed from here in PR where it is moved to installation part.